### PR TITLE
fix: fix test function

### PR DESCRIPTION
**Pull request by ChiefHolland**
# Contributor Comments

This PR adds a dependency required by the test task.

## Pull Request Checklist

Thank you for submitting a contribution to ReleaseHound!

In order to streamline the review of your contribution we ask that you review
and comply with the below requirements:

- [x] Rebase your branch against the latest commit of the target branch.
- [x] If you are adding a dependency, please explain how it was chosen.
- [x] If manual testing is needed in order to validate the changes, provide a testing plan and the expected results.
- [x] If there is an issue associated with your Pull Request, link the issue to the PR.
- [x] Validate that documentation is accurate and aligned to any project updates or additions.

### CODE REVIEW

**Review by JonZeolla — CHANGES_REQUESTED** [review]
I don't understand why we need this now but didn't before.  If it is an indirect dep it should get fixed by just freezing the lock file?

**Comment by ChiefHolland** [comment]
I was confused, too. It passed during the PR but failed the merge.

File: tasks.py [hunk 1]
```diff
@@ -151,16 +151,16 @@ def test(_c, debug=False):
...
             [
                 "pipenv",
                 "run",
                 "pytest",
                 "--cov=release_hound",
                 "tests",
             ],
-            capture_output=True,
-            check=True,
+            stdout=subprocess.PIPE,
```
Comment (ChiefHolland) [inline tasks.py:162]: Shifted subprocess method IAW https://github.com/orgs/community/discussions/26197

Comment (JonZeolla) [reply tasks.py]: WHAT 😮

**Comment by JonZeolla** [comment]
This is obsoleted by https://github.com/SeisoLLC/ReleaseHound/pull/5

**Pull request closed without merging**